example/service/s3/getObjectWithProgress/getObjectWithProgress.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"regexp"
+	"github.com/aws/aws-sdk-go/aws/credentials"
```

Comment:
Nit, would you mind running goimport on this file. It will help sort the imports into two groups standard library, and external. The imports are also sorted alphabetically.

Reply:
Done! 👍